### PR TITLE
Correct loss overwriting itself in `test` function in tutorial 'text_sentiment_ngrams'

### PR DESCRIPTION
**Pull request by etiennekintzler**
+ harmonize fields
+ remove unused `enumerate` in `train` function

### CODE REVIEW

File: beginner_source/text_sentiment_ngrams_tutorial.py [hunk 1]
```diff
@@ -187,21 +187,21 @@ def train_func(sub_train_):
...
-    acc = 0
+    test_loss = 0
+    test_acc = 0
     data = DataLoader(data_, batch_size=BATCH_SIZE, collate_fn=generate_batch)
     for text, offsets, cls in data:
         text, offsets, cls = text.to(device), offsets.to(device), cls.to(device)
         with torch.no_grad():
             output = model(text, offsets)
             loss = criterion(output, cls)
-            loss += loss.item()
```
Comment (etiennekintzler) [inline beginner_source/text_sentiment_ngrams_tutorial.py:201]: Incorrect test error since it overwrites itself at each batch

**Comment by etiennekintzler** [comment]
Hi @brianjo ! I see you're the one merging the PR 😃 

Do you have any opinion on this PR ?

**Comment by etiennekintzler** [comment]
Do you have any opinion on this PR @zhangguanheng66 

Thanks !
